app/controllers/api/cases_controller.rb

```diff
@@ -0,0 +1,51 @@
+class API::CasesController < ApplicationController
+  respond_to :json
```

Comment:
add `before_action :authenticate_user!` above this line (see other API controllers for reference)